.gitignore

```diff
@@ -51,7 +51,6 @@ Base.rte/Settings.ini
 Benchmark.rte/
 Scenes.rte/
 Metagames.rte/
-DedicatedServer.bat
```

Comment:
You'll actually wanna undo this gitignore change. That way people's local changes to the dedicated server file will be ignored by git, but we'll still have the version in the repo.